template/.github/workflows/build.yml.j2

```diff
@@ -376,4 +425,4 @@ jobs:
           ARCH_FOR_PREFLIGHT="$(arch | sed -e 's#x86_64#amd64#' | sed -e 's#aarch64#arm64#')"
           ./preflight-linux-amd64 check container "$IMAGE_TAG" --platform "${ARCH_FOR_PREFLIGHT}" > preflight.out
```

Comment:
Still confused about the `amd64` suffix, but if it works...

Reply:
This is not selecting the image, it's a hint for `./preflight-linux-amd64` which architecture it's checking on. However, since this is running only on amd64 it will only check this at this moment. @adwk67 would you rather like to see this running on `arm64` and `amd64`?

Reply:
I don't think it is that important to have preflight running on different platforms, but rather just to make sure that preflight works consistently, and the platform flag just makes sure that preflight pulls an image that matches the architecture where it is running (although it is also seems to default to the current platform _anyway_).